weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterPanel.tsx

```diff
@@ -21,7 +21,7 @@ type FilterPanelProps = {
 
 export const FilterPanel = (props: FilterPanelProps) => {
   return (
-    <div className="min-w-90 flex-auto self-stretch overflow-hidden">
+    <div className="min-w-90 flex-auto self-stretch">
```

Comment:
This minor change allows the Filter button to not disappear when the main view is very small